projects/aca-content/src/lib/components/search/search-results/search-results.component.ts

```diff
@@ -202,7 +212,14 @@ export class SearchResultsComponent extends PageComponent implements OnInit {
     this.columns = this.extensions.documentListPresets.searchResults || [];
 
     if (this.route) {
-      this.route.queryParams.pipe(takeUntil(this.onDestroy$)).subscribe((params: Params) => {
+      this.savedSearchesService
```

Comment:
do we want to do this only once with first()? What if we switch on navbar to different save search?

Reply:
I'm trying to think which solution will be best here

Reply:
I found a fix for it, you can check out the last commit